doc-content/shared-kie-docs/src/main/asciidoc/Workbench/Installation/business-central-system-properties-ref.adoc

```diff
@@ -66,13 +63,44 @@ If you plan to use RSA or any algorithm other than DSA, make sure you set up you
-endif::[]
\ No newline at end of file
+NOTE: Use this property for {CENTRAL} only. If you share a runtime environment with any other component, isolate the configuration and apply it only to {CENTRAL}.
+
+* `org.uberfire.gzip.enable`: Enables or disables Gzip compression on GzipFilter. Default value: `true`
+* `org.kie.workbench.profile`: Selects the {CENTRAL} profile. Possible values are `FULL` or `PLANNER_AND_RULES`. A prefix `FULL_` will set the profile and hide the profile preferences from the admin preferences. Default value: `FULL`
+* `org.appformer.m2repo.url`: {CENTRAL} uses the default location of the Maven repository when looking for dependencies. It points to the Maven repository inside the Workbench for example, `http://localhost:8080/business-central/maven2`. Set this property before starting up the Workbench. Default value: File path to the inner `m2` repository.
+* `appformer.ssh.keystore`: Defines the custom SSH keystore to be used with {CENTRAL} by specifying a class name. If the property is not available, the default SSH keystore is used.
+* `appformer.ssh.keys.storage.folder`: When using the default SSH keystore, this property defines the storage folder for the user’s SSH public keys. If the property is not available, the keys are stored in the Workbench `.security` folder.
+* `appformer.experimental.features`: Enables the Experimental Features Framework.
+* `org.kie.demo`: Enables external clone of a demo application from GitHub.
+* `org.kie.workbench.controller`: The URL is used to connect with the {CONTROLLER}. For example, `ws://localhost:8080/kie-server-controller/websocket/controller`.
+* `org.kie.workbench.controller.user`: The user name is used to connect with the {CONTROLLER}. Default value: kieserver
+* `org.kie.workbench.controller.pwd`: The password is used to connect with the {CONTROLLER}. Default value: kieserver1!
+* `org.kie.workbench.controller.token`: The token string is used to connect with the {CONTROLLER}.
++
+ifdef::DROOLS,JBPM[]
+NOTE: For more information about how to use token based authentication, see xref:usingTokenBasedAuthentication[].
+endif::[]
+
+* `kie.keystore.keyStoreURL`: The URL is used to load a Java Cryptography Extension KeyStore (JCEKS). For example, file:///home/kie/keystores/keystore.jceks.
+* `kie.keystore.keyStorePwd`: The password is used for the JCEKS.
+* `kie.keystore.key.ctrl.alias`: The alias of the key for default REST {CONTROLLER}.
+* `kie.keystore.key.ctrl.pwd`: The password of an alias for default REST {CONTROLLER}.
+* `org.jbpm.wb.forms.renderer.ext`: Switches the form rendering between {CENTRAL} and {KIE_SERVER}. By default, the form rendering is done by {CENTRAL}. Default value: `false`.
+* `org.jbpm.wb.forms.renderer.name`: This property allows to switch between {CENTRAL} and {KIE_SERVER} rendered forms. Default value: `workbench`.
++
+ifdef::DROOLS,JBPM[]
+To change one of these system properties in a *WildFly* or *JBoss EAP* cluster, use the following steps:
```

Comment:
I have doubts if we should keep this part. I'd like to Eder to take a look here.